network/p2p/topic_validator_test.go

```diff
@@ -52,7 +52,7 @@ func TestAuthorizedSenderValidator_Unauthorized(t *testing.T) {
 	})
 	logger := zerolog.New(os.Stdout).Level(zerolog.WarnLevel).Hook(hook)
 
-	authorizedSenderValidator := validator.AuthorizedSenderValidator(logger, channel, func(pid peer.ID) (*flow.Identity, bool) {
+	authorizedSenderValidator := validator.AuthorizedSenderValidator(logger, channel, cborcodec.NewCodec(), func(pid peer.ID) (*flow.Identity, bool) {
```

Comment:
maybe we can have a `unittest.Codec()` similar to what we have as `unittest.Logger()` currently. This also gives us the luxury of running all relevant tests against different codecs if needed.

Reply:
https://github.com/onflow/flow-go/pull/2491/commits/e1940b49cd0e78d3eb9bee9ee2aeba9a9069da06